pegasus/helpers/section_api_helpers.rb

```diff
@@ -372,6 +381,17 @@ def to_hash()
     }
   end
 
+  def to_member_hash()
```

Comment:
Still think this would be clearer with a merge so it's clearer what are the additional owner-accessible properties

def to_owner_hash
  to_member_hash.merge (
    course: course,
    teachers: teachers,
    students: students
  )
end